### DISCUSSION TITLE
Using modals ?

### DISCUSSION THREAD

**Discussion by loloof64** [DioxusLabs/dioxus#360] [Q&A] [answered]
Is it possible to use modals (such as javascript alert()) in Dioxus ?

**Comment by loloof64** [comment, marked as answer]
In fact, I've been told in a forum, that HTML has a <dialog> element. So I'll use it.

**Reply by jkelleyrtp** [reply]
We don't have a dedicated modal library yet, but you can definitely do things like `alert`. Take a look at WebSys/JsSys/Gloo.

Gloo has a "dialogs" crate that provides Rust wrappers over common dialogs.

https://github.com/rustwasm/gloo/tree/master/crates/dialogs

**Reply by loloof64** [reply]
Thank you very much. I'm gonna try it.
